components/DeckPreview.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link'
+
+import { rarityColorKey } from '../lib/utils'
+import { IDeck } from '../lib/types'
+import * as CSS from 'csstype'
+
+const containerStyle: CSS.Properties = {
+  position: "relative",
+};
+
+const deckIdStyle: CSS.Properties = {
+  position: "absolute",
+  right: "20px",
+  top: "20px",
+};
+
+const DeckPreview = ({ deck, images }: { deck: IDeck, images: any }) => {
```

Comment:
nit: we can probably import `images` directly in this file instead of passing as prop

Reply:
passing a prop made more sense to me since i wanted to keep this component as pure as possible – sticking to presentation only without importing `images` directly in this file

Reply:
Got it, sounds good! 🚀🚀🚀